lib/senders/templates/post_verify_secondary.txt

```diff
@@ -0,0 +1,10 @@
+{{t "Secondary email added" }}
+
+{{{t "You have successfully verified %(secondaryEmail)s as a secondary email to your Firefox Account. Security notifications and sign-in confirmations will now be delivered to both email addresses." }}}
+
+{{t "Manage account:"}} {{{ link }}}
+
+{{t "This is an automated email; if you received it in error, no action is required."}} {{{t "For more information, please visit %(supportUrl)s"}}}
+
+Mozilla. 331 E Evelyn Ave, Mountain View, CA 94041
+{{t "Mozilla Privacy Policy" }} {{{privacyUrl}}}
```

Comment:
Just noting that I haven't reviewed any of the email templates here at this stage...